server/controllers/events/events_controller.go

```diff
@@ -572,6 +565,14 @@ func (e *VCSEventsController) handleCommentEvent(logger logging.SimpleLogging, b
 		}
 	}
 
+	// It's a comment we're gonna react to, so add a reaction.
+	if e.EmojiReaction != "" {
```

Comment:
@marcusramberg can we add a test for this?

Reply:
I'm not quite sure how because of all the mocking. Do you have suggestions besides the existing test for reactions? Was thinking I could add a test with @bot nick but it's gonna pass regardless I guess.

Reply:
See https://github.com/runatlantis/atlantis/blob/621db395e6ada1523cdf55aedc430784a29a53e5/server/controllers/events/events_controller_test.go

Perhaps run a new test that has a comment (1) with an EmojiReaction and a test that has a comment (2) that doesn't?

Reply:
There's already a test that has an emojireaction. I've added another test now to confirm that invalid comments do not trigger the reaction method.